pwsh/buildecm.ps1

```diff
@@ -1,25 +1,30 @@
 #!/usr/bin/env pwsh
 
+$qtVersion = [version](qmake -query QT_VERSION)
+
+$kfGitRef = $args[0]
+
 # Clone
 git clone https://invent.kde.org/frameworks/extra-cmake-modules.git
 cd extra-cmake-modules
-git checkout $args[0]
+git checkout $kfGitRef
 
 # vcvars on windows
 if ($IsWindows) {
     & "$env:GITHUB_WORKSPACE/pwsh/vcvars.ps1"
 }
 
+$argQt6 = $qtVersion.Major -eq 6 ? '-DBUILD_WITH_QT6=ON' : $null
 $argDeviceArchs = $IsMacOS -and $env:buildArch -eq 'Universal' ? '-DCMAKE_OSX_ARCHITECTURES=x86_64;arm64' : $null
 
 # Build
-cmake -G Ninja . $argDeviceArchs
+cmake -G Ninja -DCMAKE_INSTALL_PREFIX="$PWD/installed" -DCMAKE_BUILD_TYPE=Release $argQt6 $argDeviceArchs .
```

Comment:
This wasn't passing in `-DBUILD_WITH_QT6=ON` even though it should have been, it somehow worked without it but I believe starting KDE Frameworks 6.8 (judging by the commits in my fork) they changed the CMake scripts and this was required for it to build successfully. Same thing for the install prefix, I forget the details but a change was needed to keep it building successfully.